src/components/Faucet/index.tsx

```diff
@@ -111,78 +136,29 @@ const FaucetPage = (): ReactElement => {
     faucetTokenRequest
   ])
 
-  useEffect(() => {
-    const fetchAddressAndNetwork = async () => {
-      try {
-        const provider = new ethers.providers.Web3Provider(window.ethereum)
-        const signer = provider.getSigner()
-        const address = await signer.getAddress()
-        const network = await provider.getNetwork()
-        setAddress(address)
-        setNetwork(networkNameMap[network.chainId] || 'Unknown')
-      } catch (error) {
-        LoggerInstance.error(error)
-      }
-    }
-
-    fetchAddressAndNetwork()
-
-    if (window.ethereum) {
-      window.ethereum.on('accountsChanged', fetchAddressAndNetwork)
-      window.ethereum.on('chainChanged', fetchAddressAndNetwork)
-    }
-
-    return () => {
-      if (window.ethereum) {
-        window.ethereum.removeListener(
-          'accountsChanged',
-          fetchAddressAndNetwork
-        )
-        window.ethereum.removeListener('chainChanged', fetchAddressAndNetwork)
-      }
-    }
-  }, [])
-
   return (
     <div className={styles.card}>
-      <h2 className={styles.title}>Welcome to the Pontus-X Faucet</h2>
-      <p className={styles.description}>
-        A faucet is a service that provides free tokens for testing purposes.
-        Known accounts can request 10 EUROe fee tokens and 1000 EUROe payment
-        tokens to use on the Pontus-X network. These tokens are available every
-        12 hours.
-      </p>
+      <h2 className={styles.title}>{cardTitle}</h2>
+      <p className={styles.description}>{cardDescription}</p>
       <div className={styles.instructions}>
-        <h3>How to Request Tokens:</h3>
+        <h3>{cardExplainerTitle}</h3>
         <ol>
-          <li>Ensure you have a web3 wallet (e.g. MetaMask) connected.</li>
-          <li>
-            Your wallet address will be automatically detected and displayed
-            below.
-          </li>
-          <li>
-            Click the &quot;Get Tokens&quot; button to request your free tokens.
-          </li>
-          <li>Wait for a few seconds while the request is processed.</li>
-          <li>
-            You will receive a confirmation once the tokens are successfully
-            added to your wallet.
-          </li>
+          <li>{cardExplainerFirstStep}</li>
+          <li>{cardExplainerSecondStep}</li>
+          <li>{cardExplainerThirdStep}</li>
+          <li>{cardExplainerFourthStep}</li>
+          <li>{cardExplainerFithStep}</li>
         </ol>
-        <p>
-          <strong>Note:</strong> You can only request tokens once every 12
-          hours. If you encounter any issues, please try again later.
-        </p>
       </div>
       <div className={styles.address}>
-        <strong>{label}:</strong> {address}
+        <strong>{cardNetworkAddress}:</strong> {accountAddress}
       </div>
       <div className={styles.network}>
-        <strong>Connected Network:</strong> {network}
+        <strong>{cardNetwork}:</strong> {network}
```

Comment:
suggestion: we should be able to use the `<NetworkName>` component here and remove a bit of duplication.